src/LicenseTextReader.ts

```diff
@@ -85,7 +85,10 @@ class LicenseTextReader {
   ): string | null {
     for (const path of paths) {
       const filePath = this.fileSystem.join(modulePath, path);
-      if (/^licen[cs]e/i.test(path) && !this.fileSystem.isDirectory(filePath)) {
+      if (
+        /\blicen[cs]e\b/i.test(path) &&
+        !this.fileSystem.isDirectory(filePath)
+      ) {
```

Comment:
I think it would be safer to keep the original regex first, and if nothing is found from the first regex, then use the other regex.

Reply:
That makes sense, thanks - change applied (including a misstep where I considered moving the `isDirectory` test; that could be relatively expensive on some systems I suppose).